src/cascadia/TerminalApp/TerminalPage.xaml

```diff
@@ -68,6 +68,20 @@ the MIT License. See LICENSE in the project root for license information. -->
             DefaultButton="Primary">
         </ContentDialog>
 
+        <ContentDialog
+            x:Load="False"
+            x:Name="UnsupportedSchemeDialog"
+            x:Uid="UnsupportedSchemeDialog"
+            DefaultButton="Primary">
```

Comment:
should the unsupported scheme dialog have an "Open Anyway" button that will allow the user an escape hatch?

Reply:
I have to say that makes me quite nervous - I am not sure what ShellExecute can do with other schemes and we do not have anything in place yet to check for safety of the link

Reply:
You could show the full link in the dialog box though and allow the user to copy it, so they can get the data. There could even be a button called "Copy link to clipboard".

Reply:
I agree that there should be *some* way of letting the user take the URI out if we're not going to handle it ourselves for whatever reason.

Reply:
The user can copy the URI from the dialog box that pops out! (Its selectable)